tbtacl/CMakeLists.txt

```diff
@@ -1,4 +1,5 @@
 set(TBTACL "tbtacl")
+set(TBTXDOMAIN "tbtxdomain")
```

Comment:
Isn't it better to keep it at a separated dir?

Reply:
Sure, I can move it to separate dir. How about udev rules, should I make a separate rule that then gets installed as 60-tbtxdomain.rules or do you think it is better to merge them?

Reply:
If we prefer to separate the scripts to separated dirs, the udev rules should be separated too.
But maybe the udev rules are a reason to keep them together?

Reply:
Right that was the reason I put it originally to the same directory. However, I think it makes more sense to have them in a separate directories and have separate udev rules. I'll update the patch accordingly.